.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "8"
+
+before_script:
+  - ./node_modules/.bin/testrpc -h '0.0.0.0'&
+  - export TRUFFLE_PID=$!
+
+after_script:
+  - kill $TRUFFLE_PID
```

Comment:
do we need to clean up? as far as i know travis launches a docker container for every build and then throws it away. would not mention this if it was a jenkins script or sth.

Reply:
not sure, let me check